<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalGenerator</class>
 <widget class="QWidget" name="SignalGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>单频波形参数设置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0" columnminimumwidth="0,0,0">
        <item row="2" column="0">
         <widget class="QLabel" name="ampLbl">
          <property name="text">
           <string>峰峰值</string>
          </property>
          <property name="buddy">
           <cstring>ampBox</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="waveformLbl">
          <property name="text">
           <string>波形</string>
          </property>
          <property name="buddy">
           <cstring>waveformBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="freqStepLbl">
          <property name="text">
           <string>频率步进</string>
          </property>
          <property name="buddy">
           <cstring>freqStepBox</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="freqLbl">
          <property name="text">
           <string>频率值</string>
          </property>
          <property name="buddy">
           <cstring>freqBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;</string>
          </property>
          <property name="text">
           <string>设置参数值</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QSpinBox" name="freqStepBox">
          <property name="suffix">
           <string> Hz</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="ampBox">
          <property name="suffix">
           <string> V</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>4.400000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>4.400000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QSpinBox" name="freqBox">
          <property name="suffix">
           <string> Hz</string>
          </property>
          <property name="maximum">
           <number>16777215</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QComboBox" name="waveformBox">
          <item>
           <property name="text">
            <string>正弦波</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>方波</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>三角波</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="0" colspan="3">
         <widget class="QPushButton" name="setValueBtn">
          <property name="text">
           <string>设置模块参数值</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="3">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;</string>
          </property>
          <property name="text">
           <string>当前参数值</string>
          </property>
          <property name="buddy">
           <cstring>curValueText</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="curValueText">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="readValueBtn">
          <property name="text">
           <string>读取模块参数值</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>扫频参数设置</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
        <item row="0" column="1">
         <widget class="QSpinBox" name="lwrFreqBox">
          <property name="suffix">
           <string> Hz</string>
          </property>
          <property name="maximum">
           <number>16777215</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="scanPeriodLbl">
          <property name="text">
           <string>扫频时隙</string>
          </property>
          <property name="buddy">
           <cstring>scanPeriodBox</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="scanPeriodBox">
          <property name="suffix">
           <string> ms</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="scanFreqStepLbl">
          <property name="text">
           <string>频率步进</string>
          </property>
          <property name="buddy">
           <cstring>scanFreqStepBox</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="scanFreqStepBox">
          <property name="suffix">
           <string> Hz</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lwrFreqLbl">
          <property name="text">
           <string>频率下限</string>
          </property>
          <property name="buddy">
           <cstring>lwrFreqBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="uprFreqLbl">
          <property name="text">
           <string>频率上限</string>
          </property>
          <property name="buddy">
           <cstring>uprFreqBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="uprFreqBox">
          <property name="suffix">
           <string> Hz</string>
          </property>
          <property name="maximum">
           <number>16777215</number>
          </property>
          <property name="value">
           <number>100000</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="scanOnceRadio">
          <property name="text">
           <string>单次扫频</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="scanLoopRadio">
          <property name="text">
           <string>循环扫频</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="scanBtn">
          <property name="text">
           <string>启动扫频</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>freqStepBox</tabstop>
  <tabstop>ampBox</tabstop>
  <tabstop>freqBox</tabstop>
  <tabstop>waveformBox</tabstop>
  <tabstop>setValueBtn</tabstop>
  <tabstop>curValueText</tabstop>
  <tabstop>readValueBtn</tabstop>
  <tabstop>lwrFreqBox</tabstop>
  <tabstop>uprFreqBox</tabstop>
  <tabstop>scanFreqStepBox</tabstop>
  <tabstop>scanPeriodBox</tabstop>
  <tabstop>scanOnceRadio</tabstop>
  <tabstop>scanLoopRadio</tabstop>
  <tabstop>scanBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
